packages/angular_devkit/build_angular/test/browser/build-optimizer_spec_large.ts

```diff
@@ -21,7 +21,7 @@ describe('Browser Builder build optimizer', () => {
 
   it('works', (done) => {
     const overrides = { aot: true, buildOptimizer: true };
-    runTargetSpec(host, browserTargetSpec, overrides).pipe(
+    runTargetSpec(host, browserTargetSpec, overrides, DefaultTimeout * 2).pipe(
```

Comment:
With TypeScript 3, build optimiser is taking more time to complete. For this spec takes `28 secs` in TS2.9 while `36 secs` in TS3 (locally)